apps/chat/src/backend/serverless/template.yaml

```diff
@@ -697,6 +800,286 @@ Resources:
              body: 'Not Authorized'
            };
          };
+
+  ###
+  ### RESOURCES FOR MEETING SERVER
+  ###
+
+  CreateMeetingApiFunction: # Adds a POST api endpoint at "/create" to create meetings
+    Type: AWS::Serverless::Function
+    Properties:
+      Role: !GetAtt AuthLambdaIAMRole.Arn
+      Events:
+        ApiEvent:
+          Type: Api
+          Properties:
+            Path: /create
+            Method: POST
+            RestApiId:
+              Ref: ApiGatewayApi
+      Environment:
+        Variables:
+          ChimeAppInstanceArn: !GetAtt TriggerChimeAppInstanceLambda.AppInstanceArn
+      Layers:
+        - !Ref AWSSDKChimeLayer
+      Runtime: nodejs12.x
+      Handler: index.handler
+      InlineCode: |

```

Comment:
For the sake of the blog and keeping all the sources up-to-date, unless we switch to CDK, we would have to upload these to an S3 bucket and reference them in the CF. You're absolutely right that as separate files, we would be able to better run formatters and read the code but for the deployment, it would make keeping a consistent version for the CF templates and lamdbda handlers more difficult.